src/frontend/js/api/lms/openedx-fonzie.ts

```diff
@@ -87,6 +90,30 @@ const API = (APIConf: AuthenticationBackend | LMSBackend): APILms => {
 
           throw new HttpError(responseError.status, responseError.statusText);
         },
+        update: async (username: string, data: OpenEdxFullNameFormValues) => {
+          /*
+            edx_csrf_token cookie is set through a SET_COOKIE header send from a previous request
+            e.g: Getting user enrollment set the cookie
+          */
+          // Ensure CSRF is fresh
+          await fetch(APIOptions.routes.setCsrfToken, { credentials: 'include' });
```

Comment:
This is useless right now. The fonzie route to retrieve user information now sets the CSRF cookie.

```suggestion
```